cmd/machine-config-daemon/start.go

```diff
@@ -177,6 +177,7 @@ func runStartCmd(_ *cobra.Command, _ []string) {
 		ctrlctx.InformerFactory.Machineconfiguration().V1().MachineConfigs(),
 		ctrlctx.KubeInformerFactory.Core().V1().Nodes(),
 		ctrlctx.InformerFactory.Machineconfiguration().V1().ControllerConfigs(),
+		ctrlctx.KubeInformerFactory.Core().V1().ConfigMaps(),
```

Comment:
Can  we restrict this to just `openshift-config-managed`?  For example, [in the CVO, we have][1]:

```go
openshiftConfigManagedInformer := informers.NewSharedInformerFactoryWithOptions(kubeClient, resyncPeriod(o.ResyncInterval), informers.WithNamespace(internal.ConfigManagedNamespace))
```

I'm not actually sure if that filtering is performed client-side or server-side.  Would be nice if someone who understood informers better had some context :)

[1]: https://github.com/openshift/cluster-version-operator/blob/7c22805ee23be586dcff97c673ca648f1810770b/pkg/start/start.go#L482

Reply:
yeah we can do `NamedspacedKubeInformerFactory` I think. This kind of solves the above clusterrole issue.